src/playground/app.json

```diff
@@ -12,6 +12,11 @@
     "orientation": "portrait",
     "packagerOpts": {
       "config": "./metro.config.js"
+    },
+    "androidStatusBarColor": "#3366FF",
```

Comment:
what about ios?

Reply:
I guess we don't need specify this for ios.